### PR TITLE
add note about possible differences in date output

### PR DESCRIPTION
**Pull request by Graziel**

### CODE REVIEW

File: en/core-libraries/time.rst [hunk 1]
```diff
@@ -180,6 +180,11 @@ The following calendar types are supported:
 
 .. versionadded:: 3.1
     Non-gregorian calendar support was added in 3.1
+    
+.. note::
+    For constant strings (ie IntlDateFormatter::FULL) Intl uses ICU library
```
Comment (ravage84) [inline en/core-libraries/time.rst]: ``i.e.``
https://en.wiktionary.org/wiki/i.e.

I would go with:
> For constant strings, i.e. ``IntlDateFormatter::FULL``, Intl uses ICU...

File: en/core-libraries/time.rst [hunk 2]
```diff
@@ -180,6 +180,11 @@ The following calendar types are supported:
 
 .. versionadded:: 3.1
     Non-gregorian calendar support was added in 3.1
+    
+.. note::
+    For constant strings (ie IntlDateFormatter::FULL) Intl uses ICU library 
+    that feeds its data from CLDR (http://cldr.unicode.org/) which version
```
Comment (ravage84) [inline en/core-libraries/time.rst]: Link CLDR to http://cldr.unicode.org/, instead.

Comment (Graziel) [inline en/core-libraries/time.rst]: @ravage84 link to http://cldr.unicode.org/ instead of http://cldr.unicode.org/ ? but they are the same link :)

Comment (ravage84) [inline en/core-libraries/time.rst]: http://unicode.org/

Comment (Graziel) [inline en/core-libraries/time.rst]: not sure how this link would help clarify what is CLDR, IMHO it only makes user harder to find that info

**Pull request merged**